### PR TITLE
don't reassign the same cases multiple times in the same form

### PR DESCRIPTION
**Pull request by czue**
http://manage.dimagi.com/default.asp?188322

Hoping this helps make some progress on all the "aborting because of doc update" errors on the care-bihar domain.

This workflow is terrible, but trying to improve it with band-aids rather than doing something bigger. According to Matt we can likely get rid of this code in about 6 months.

### CODE REVIEW

**Comment by czue** [comment]
@benrudolph or whoever

**Comment by snopoke** [comment]
:+1: waiting for build

File: custom/bihar/signals.py [hunk 1]
```diff
@@ -24,17 +24,21 @@ def bihar_reassignment(sender, xform, cases, **kwargs):
         else:
             [owner_id] = owner_ids
             if owner_id not in DEMO_OWNER_IDS:
-                form_cases = set(c._id for c in cases)
-
+                # don't attempt to reassign the cases included in this form
+                cases_not_to_touch = set(c._id for c in cases)
```
Comment (benrudolph) [inline custom/bihar/signals.py]: way better name

**Comment by czue** [comment]
:white_check_mark:

**Pull request merged**